js_modules/dagster-ui/packages/ui-core/src/workspace/WorkspaceContext/WorkspaceStatusPoller.ts

```diff
@@ -68,6 +68,9 @@ export class WorkspaceStatusPoller {
   }
 
   private async loadFromServer() {
```

Comment:
This function is called in a `setInterval` so if we skip this call its fine because it will make another one in 5 seconds